graylog2-web-interface/src/components/navigation/InactiveNavItem.jsx

```diff
@@ -4,21 +4,19 @@ import { NavItem } from 'components/graylog';
 
 // Don't pass active prop, since NavItem should always be inactive
 // eslint-disable-next-line no-unused-vars
-function InactiveNavItem({ active, children, ...props }) {
-  return <NavItem {...props}>{children}</NavItem>;
+function InactiveNavItem({ active, className, children, ...props }) {
+  return <NavItem className={className} {...props}>{children}</NavItem>;
 }
 
 InactiveNavItem.propTypes = {
   active: PropTypes.any,
-  children: PropTypes.oneOfType([
-    PropTypes.array,
-    PropTypes.element,
-    PropTypes.string,
-  ]).isRequired,
+  children: PropTypes.oneOfType([PropTypes.array, PropTypes.element, PropTypes.string]).isRequired,
+  className: PropTypes.string,
```

Comment:
Could you please explain why we made this change here? I don't see where it's being used.

Reply:
The gist is that styled-components passes its styles via `className` so in order to do something like `styled(Component)` that Component needs to accept the `className` prop

https://styled-components.com/docs/basics#styling-any-component

Reply:
>  I don't see where it's being used.

`src/components/navigation/NotificationBadge.jsx:14`

Reply:
Ok, it makes sense now, thanks!